eventuals/loop.h

```diff
@@ -398,10 +398,10 @@ struct _Loop final {
           std::move(stop_));
     }
 
-    template <typename Error = std::exception, typename... Errors>
+    template <typename... Errors>
     auto raises() && {
```

Comment:
For consistency let's add the version of `raises<std::tuple<Error1, Error2, ...>>()` here just like we have with `Eventual`. And let's also add in `stream.h` too.